docs/website/content/docs/installation.md

```diff
@@ -7,17 +7,20 @@ draft: false
 bref: "Sysl can be installed on Windows, MacOS and Linux - follow this guide"
 toc: true
 ---
-Sysl is a CLI (Command Line Interface) that excecutes with the `sysl` command. 
 
-`go get -v github.com/anz-bank/sysl/cmd/sysl`
+Sysl is a CLI (Command Line Interface) that executes with the `sysl` command.
```

Comment:
This still needs to be reworded imo - i'll leave to the native speakers if they agree.

Reply:
It needs to be re-worded, but I'm not quite sure what it needs to say. Is Sysl more than just the cli? is some library you link against? or is the `sysl` CLI the only entrypoint into SYSL?

Reply:
If you want you can say it is also a programming language - right?
And as such there have been various IDE/automation attempts around it -  e.g. build it into a website or so.
It is also intended to be "easily extensible" I think - like `git-pluginname` or so, not sure if any of that has happened yet.
I'll just leave it for now.